spec.bs

```diff
@@ -1347,14 +1420,19 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|:
   1. Set |config| to |leadingBidInfo|'s [=leading bid info/auction config=].
   1. Let |sellerReportingScript| be the result of [=fetching script=] with |config|'s
     [=auction config/decision logic url=].
-  1. Let « |sellerSignals|, |reportUrl|, |reportingBeaconMap| » be the result of [=evaluating a
-     reporting script=] with |sellerReportingScript|, "`reportResult`", and « |config|,
-     |browserSignals|».
-  1. TODO: Store |reportUrl| and |reportingBeaconMap| in the {{FencedFrameConfig}} as appropriate.
+  1. Let |reportResultOutputJS| be the result of [=evaluating a reporting script=] with
+     |sellerReportingScript|, "`reportResult`", and « |config|, |browserSignals|».
+  1. If |reportResultOutputJS| is an [=ECMAScript/abrupt completion=], [=iteration/continue=].

```

Comment:
As you pointed out in my comment below, this isn't in a loop, so it should return instead of continuing.